src/pages/index.js

```diff
@@ -86,26 +178,29 @@ function getCardElement(data) {
   cardImageEl.src = data.link;
   cardImageEl.alt = data.name;
 
-  cardLikeBtn.addEventListener("click", () => {
-    cardLikeBtn.classList.toggle("card__like-button_liked");
-  });
-
-  cardTrashBtn.addEventListener('click', handleCardDelete);
+  if (data.isLiked) {
+    cardLikeBtn.classList.add("card__like-button_liked");
+  }
 
-  cardImageEl.addEventListener("click", () => {
-    previewModalImageEl.src = data.link;
-    previewModalImageEl.alt = data.name;
-    previewModalCaptionEl.textContent = data.name;
-    openModal(previewModal);
-  });
+  cardLikeBtn.addEventListener("click", (evt) => handleLike(evt, data._id));
+  cardTrashBtn.addEventListener("click", () =>
+    handleCardDelete(cardElement, data._id)
+  );
+  cardImageEl.addEventListener("click", () => handleImageClick(data));
 
   return cardElement;
 }
 
-const handleCardDelete = (evt) => {
-  evt.target.closest(".card").remove();
-};
-
+function handleDeleteSubmit(evt) {
+  evt.preventDefault();
```

Comment:
*  According to the task, you need to change submit buttons text (`Saving…`) while waiting for the server response. And the user should see it.


And when you delete a card, there should be `Deleting...`

Reply:
My remark hasn't been corrected